.gitignore

```
@@ -10,3 +10,9 @@ Homestead.json
 Homestead.yaml
 npm-debug.log
 yarn-error.log
+public/js/app.js
```

This seems a little odd. I'd probably do something like this instead and not check in the generated files.

```suggestion
public/js
public/css
```